views/resume.hbs

```diff
@@ -0,0 +1,71 @@
+<header class="basics">
```

Comment:
Love the fact that you used handlebars! Did you know you can use HBS in the frontend? no need for nodejs.

Reply:
I didn't know that!